tests/shutdown/shutdown.pm

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2015 SUSE Linux Products GmbH
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+use strict;
+use base 'basetest';
+use testapi;
+
+sub run {
+    my $self = shift;
+
+    if (get_var("SAVE_IMAGE_HDD_1")) {
+        upload_image(1, get_var("SAVE_IMAGE_HDD_1"));
```

Comment:
as Ondrej's test uses STORE_ASSET, shouldn't we stick to the STORE term? 

Now STORE_ASSET is very generic and SAVE_IMAGE_HDD_1 very specific. Perhaps you can come up with something in the middle?

@sysrich - any suggestions? One is public, one is private - but that's about it IMO